.env.example

```
@@ -78,3 +78,12 @@ FEATURE_PHILOSOPHY_VALIDATION=false
 
 # Optional bootstrap semantic retrieval candidate pool size (safe bounds: 1..5000).
 FOOD_SEARCH_SEMANTIC_CANDIDATE_LIMIT=250
+
+# === Local Execution Sandbox ===
+# Optional local-only bounded executor for agent tasks on a developer machine.
+# Keep disabled by default until a sandbox root and allowlist are configured.
+AGENT_EXECUTION_SANDBOX_ENABLED=false
+AGENT_EXECUTION_SANDBOX_ROOT=.
+AGENT_EXECUTION_SANDBOX_TIMEOUT_SECONDS=30
+AGENT_EXECUTION_SANDBOX_MAX_OUTPUT_BYTES=32768
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AGENT_EXECUTION_SANDBOX_MAX_OUTPUT_BYTES key should go before the AGENT_EXECUTION_SANDBOX_ROOT key

(UnorderedKey)